app/models/foreman_tasks/concerns/action_triggering.rb

```diff
@@ -73,6 +73,7 @@ def plan_destroy_action
       # of planning phase is expected to be commited when execution occurs. Also
       # we want to be able to rollback the whole db operation when planning fails.
       def plan_action(action_class, *args)
+        return if ForemanTasks.dynflow.config.disable_active_record_actions
```

Comment:
I've put it this way as it represents better (for me) the default behaviour. It's more straight here, that the active record actions are enabled by default and you can disable it when you want.

Not strong preference about it, so I can change it if you insist :)